src/main/java/com/google/roomies/database/FirebaseDatabase.java

```diff
@@ -61,6 +64,38 @@ public void addListingAsMap(Listing listing) {
     this.db.collection(LISTING_COLLECTION_NAME).add(data);
   }
 
+  /**
+  * Add a comment to a collection as a map.
```

Comment:
For the override methods, javadoc is already being added in interface. Can be omitted in the implementation.

Reply:
Done